### PR TITLE
Wrote a script that generates a kubeconfig for a serviceaccount

### PR DESCRIPTION
**Pull request by isaacdorfman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by awels** [comment]
If you could git commit --amend -s to sign the PR, that would be great

**Comment by brybacki** [comment]
@isaacdorfman that is nice, 
@davidvossel of course this will be useful

I was about to do something like that, but I have updated kubevirtci and I am stuck in the dependency hell. But that is a discussion for a different PR (https://github.com/kubevirt/csi-driver/pull/35 )

**Comment by brybacki** [comment]
I am using this code in https://github.com/kubevirt/csi-driver/pull/39. But still a draft.

**Comment by brybacki** [comment]
@isaacdorfman I took your idea here https://github.com/kubevirt/csi-driver/pull/39/commits/bd75953c769b7e79f210c322dbb683599dbf7b91 
If it is not a problem can we close this PR?

File: hack/create-infra-kubeconfig.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+SERVICE_ACCOUNT_NAME=$1
+
+TOKEN_NAME=$(kubectl get serviceaccount kubevirt-csi -o json | jq '.secrets[0].name' | xargs echo)
```
Comment (mfranczy) [inline hack/create-infra-kubeconfig.sh:4]: That will not work with k8s 1.24, due to https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token#kep-2799-reduction-of-secret-based-service-account-tokens

Comment (brybacki) [reply hack/create-infra-kubeconfig.sh:4]: Thanks for the comment, I have converted this to issue, so we can handle it,
This was already merged as part of another change

**Pull request closed without merging**